Sources/InputTextView.swift

```diff
@@ -84,7 +90,8 @@ open class InputTextView: UITextView {
     }
 
     required public init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+        super.init(coder: aDecoder)
+        setup()
```

Comment:
@nathantannar4 Is this part of the workaround? There was a problem with unarchiving?